.travis.yml

```diff
@@ -20,114 +20,123 @@
 
 # This is the Travis CI top-level script to setup and run the NEST build and test environment. 
 
+language: python
 
-language: minimal
-
-# Required for Ubuntu Trusty (14.04 LTS).
+# Using the container infrastructure for Ubuntu Trusty (14.04 LTS).
 sudo: required
 dist: trusty
+
 #https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2
-group: edge
+group: deprecated-2017Q2
```

Comment:
Here is a possible fix: [https://gist.github.com/dan-blanchard/7045057](https://gist.github.com/dan-blanchard/7045057)
Otherwise [this might be adapted](https://stackoverflow.com/a/17254588/5962321), I think.